tests/keras/backend/backend_test.py

```diff
@@ -1833,6 +1833,16 @@ def test_variable_support_bool_dtype(self):
             with pytest.raises(TypeError):
                 K.variable('', dtype='unsupported')
 
+    def test_clip_with_tensors(self):
+        # Create tensor
+        import keras
```

Comment:
You should not be importing `keras` here. You can use `K.placeholder` to create a placeholder.